tests/unit/controlgroup/controlgroup_methods.js

```diff
@@ -0,0 +1,130 @@
+module( "Controlgroup: methods" );
+
+test( "destroy", function() {
+	expect( 1 );
+	domEqual( ".controlgroup", function() {
+		$( ".controlgroup" ).controlgroup().controlgroup( "destroy" );
+	});
+});
+test( "disable", function() {
+	var tests = 1,
+		element = $( ".controlgroup" ).controlgroup().controlgroup( "disable" );
+	strictEqual( element.hasClass( "ui-state-disabled" ), false,
```

Comment:
Because the individual widgets, within the controlgroup, are disabled, not the controlgroup itself.